server/master/src/main/java/org/apache/accumulo/master/replication/RemoveCompleteReplicationRecords.java

```diff
@@ -216,4 +220,13 @@ protected long removeRowIfNecessary(BatchWriter bw, SortedMap<Key,Value> columns
 
     return recordsRemoved;
   }
+
+  /*
+   * Metrics calls snapshot. Largest time stamp will be found for each snapshot then reset.
+   */
+  private void collectLatency(Long createdTime) {
+    long latency = System.currentTimeMillis() - createdTime;
```

Comment:
Definitely don't use nanoTime across processes. It's only guaranteed that `(newtime - oldtime) > 0` within the same process.